src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -230,7 +241,99 @@ function initial_condition_shock_bubble(x, t, equations::CompressibleEulerMultic
     p       = p_2
   end 
 
-  return prim2cons(SVector(rho1, rho2, v1, v2, p), equations)
+  return prim2cons(SVector(v1, v2, p, rho1, rho2), equations)
+end
+
+
+function initial_condition_shock_bubble_3comp(x, t, equations::CompressibleEulerMulticomponentEquations2D)
```

Comment:
```suggestion
function initial_condition_shock_bubble(x, t, equations::CompressibleEulerMulticomponentEquations2D{3})
```
If you restrict the other method above as similarly, you can avoid introducing another name for basically the same setup (varying only for different equations). Optimally, this method should also get a docstring.

Reply:
done in 4c720b0